src/test/smoke/heatlhCheck.ts

```diff
@@ -0,0 +1,12 @@
+import request from 'supertest';
+import { app } from '../../main/app';
+import { expect } from 'chai';
+
+describe('GET /health', () => {
+  test('should return 200 and UP status', async () => {
+    await request(app)
```

Comment:
i dont think these are doing http requests here, they seem to be getting the offline output instead of checking whats there